client/src/app/models/user.js

```diff
@@ -17,7 +17,14 @@ window.User = Backbone.Model.extend({
         window.localStorage.setItem("_user.name", JSON.parse(user).username);
         router.navigate('/', { trigger: true });
       },
-      error: function(err) { console.log(err.status + " " + err.statusText); }
+      error: function(err) {
+        console.log(err.status + " " + err.statusText);
+        var options = {
+          text: "Error: Incorrect username or password. Please try again."
+        };
+        this.formErrorView = new formErrorView(options);
```

Comment:
I was trying to trigger an event that we would listen to in a view. But that would require the to be instantiated somewhere else and with the user model. maybe put instantiating the error form in the utils?